modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -159,6 +159,26 @@ public void testAddObjectRelationship() throws Exception {
 		_objectRelationshipLocalService.deleteObjectRelationship(
 			objectRelationship);
 
+		List<ObjectField> objectFields =
+			_objectFieldLocalService.getObjectFields(
+				_objectDefinition2.getObjectDefinitionId());
+
+		ObjectField objectField = objectFields.get(0);
+
+		String objectFieldName = objectField.getName();
```

Comment:
I believe will be better to understand the test if we add a new `objectField` to `objectDefinition2` and try to add a new `objectRelationship` with same name.